src/aqdata/tests/lsgc_test.cc

```diff
@@ -14,18 +30,13 @@ class LSGCTest : public ::testing::Test {
                        dealii::Patterns::Selection ("lsgc"), "");
     prm.declare_entry ("number of groups", "1",
                        dealii::Patterns::Integer (), "");
-  }
-
-  template <int dim>
-  void AQDataTest();
-  
-  dealii::ParameterHandler prm;
-};
+    prm.declare_entry ("transport model", "regular",
+                     dealii::Patterns::Selection("regular|ep"), "");
```

Comment:
Either add two more space before `dealii::Patterns...` or put it forward with only four more indent spaces (6 in total), i.e. 
```prm.declare_entry ("transport model", "regular",
    dealii::Patterns::Selection("regular|ep"), "");```

Reply:
Good eye! I clearly missed hitting tab there.